From 04ab3883276d5e35a6676b2f9ad90c1dbeb69e79 Mon Sep 17 00:00:00 2001
Message-ID: <04ab3883276d5e35a6676b2f9ad90c1dbeb69e79.1737500778.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1737500778.git.sharlatanus@gmail.com>
References: <cover.1737500778.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 15 Jan 2025 21:03:51 +0000
Subject: [PATCH 03/27] gnu: stellarium: Update to 24.4-0.60b6bec.

* gnu/packages/astronomy.scm (stellarium): Update to 24.4-0.60b6bec.
[source]: Use the latest commit containing the build fix.
[inputs]: Remove indi, add indi-2.0.

Change-Id: Ic35bb032bd56d1a00e7de5d6de57946e25325a69
---
 gnu/packages/astronomy.scm | 17 +++++++++++------
 1 file changed, 11 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 2bd8fecf754..545a7434e69 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -7127,18 +7127,23 @@ (define-public stackistry
     (license license:gpl3+)))
 
 (define-public stellarium
-  (package
+  ;; XXX fatal error: libs/indiclient/baseclient.h: No such file or directory
+  ;; <https://github.com/Stellarium/stellarium/issues/4019>, using the latest
+  ;; commit.
+  (let ((commit "60b6becd4147dd7c731ed0be790b1a30cdc0039b")
+        (revision "0"))
+    (package
     (name "stellarium")
-    (version "24.3")
+    (version (git-version "24.4" revision commit))
     (source
      (origin
        (method git-fetch)
        (uri (git-reference
              (url "https://github.com/Stellarium/stellarium")
-             (commit (string-append "v" version))))
+             (commit commit)))
        (file-name (git-file-name name version))
        (sha256
-        (base32 "0wwaddbqcia6jaz3lc1cf63pvi9bqj2cglp428i77jlfs3dfj45j"))))
+        (base32 "1i00f63pmyy55mwagab4pv419agzmg2c4075hc8cgbhhhfr6gm8d"))))
     (build-system cmake-build-system)
     ;; TODO: Complete documentation build and split into dedicated outputs.
     (arguments
@@ -7162,7 +7167,7 @@ (define-public stellarium
     (inputs
      (list calcmysky-qt5
            gpsd
-           indi
+           indi-2.0
            libnova
            nlopt
            openssl
@@ -7192,7 +7197,7 @@ (define-public stellarium
 3D, just like what you see with the naked eye, binoculars, or a telescope.  It
 can be used to control telescopes over a serial port for tracking celestial
 objects.")
-    (license license:gpl2+)))
+    (license license:gpl2+))))
 
 (define-public stuff
   (package
-- 
2.47.1

